packages/ia-components/sandbox/multimode-image-viewer/multimode-image-viewer.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
```

Comment:
Is there an issue that reflects this component?  If so, let's link to it. If not, let's make one and get our designer @pezvi involved in the conversation.